sdk/python/feast/cli.py

```diff
@@ -236,7 +236,7 @@ def materialize_incremental_command(end_ts: str, views: List[str]):
 @click.option(
     "--template",
     "-t",
-    type=click.Choice(["local", "gcp"], case_sensitive=False),
+    type=click.Choice(["local", "gcp", "aws_dynamo"], case_sensitive=False),
```

Comment:
I think we can just call it `aws` for now. Dynamo can be an implicit default.

Reply:
Sorry about this, but I think what you had here was correct. Would you mind using `aws_dynamo` until we have Redshift support?